openclaw_hooks/nima-recall-live/lazy_recall.py

```
@@ -1,468 +1,438 @@
     try:
         data = json.loads(affect_json_str)
-        if isinstance(data, dict):
-            return {k: float(v) for k, v in data.items() 
-                    if k in AFFECT_DIMENSIONS and isinstance(v, (int, float))}
-    except (json.JSONDecodeError, TypeError, ValueError):
-        pass
-    return {}
-
+        return {k: float(v) for k, v in data.items() 
+                if k in AFFECT_DIMENSIONS and isinstance(v, (int, float))}
+    except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)